content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/autopackets.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Multiplayer AutoPackets
+description: A reference for defining Open 3D Engine multiplayer packets through autopackets.
+linktitle: AutoPackets
+---
+
+*AutoPackets* provide a convenient way to define network packets used to communicate data between endpoints in a Multiplayer session. Using the [AzAutoGen](/docs/user-guide/engine/autogen) system, autopacket files found inside of your project are processed during builds to create C++ classes and handlers for packets with defined payloads.
```

Comment:
To me its not clear what an auto-packet is here? Are we defining a protocol for communication? Are they are collecting of attributes for a packet?

Could you provide an example(s) of a 'typical' autopacket here ie a Connect packet that contains the properties expected during connection events such as a protocol version and ticket id?

Reply:
There's an example detailing that further down in the doc.